euphoria-local/src/main/java/cz/seznam/euphoria/executor/local/WatermarkEmitStrategy.java

```diff
@@ -39,11 +42,6 @@ public void schedule(Runnable action) {
       scheduler.scheduleAtFixedRate(action, 100, 100, TimeUnit.MILLISECONDS);
     }
 
-    @Override
-    public void close() {
-      scheduler.shutdown();
-    }
-
```

Comment:
Why we don't need to shutdown the executor anymore?

Reply:
it is a static deamonthread executor, so it will shut down automatically when jvm stops. And nobody was using this method anyway :)